.solcover.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  skipFiles: ['mocks', 'utils/Migrations.sol'],
+  providerOptions: {
+      default_balance_ether: 1000000000,
+      total_accounts: 20
+  },
+  mocha: {
+    grep: "@skip-on-coverage",
+    invert: true
+  }
+}
```

Comment:
This config says:
+ Do not instrument or report coverage for the fake contracts (or Migrations)
+ Use the same number of accounts as the buidlerEVM run does (using ganache-cli)
+ Begin with a high balance - coverage consumes a lot of gas
+ Skip some tests that only fail with solidity-coverage